### PR TITLE
fix(minecraft-dev/celebi): add helm values

### PR DESCRIPTION
**Pull request by LooterToGo**

### CODE REVIEW

File: minecraft-dev/minecraft/minecraft-helm-values-celebi.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+extraEnv:
+  CFG_GTS_DISCORD_WEBHOOK: "$MC_GTS_DISCORD_WEBHOOK"
+  CFG_MASK_APIKEY: "$MC_MASK_APIKEY"
+  CFG_NUVOTE_TOKEN: "$MC_NUVOTE_TOKEN"
+  CFG_PIXELMONOPTIMIZER_PASSWORD: "$MC_PIXELMONOPTIMIZER_PASSWORD"
+  CFG_TEBEX_KEY: "$MC_TEBEX_KEY"
+  GENERIC_PACKS_PREFIX: "$MC_DEV_PACK_URL"
```
Comment (ChipWolf) [inline minecraft-dev/minecraft/minecraft-helm-values-celebi.txt:7]: Only the Tebex env var should be in here

**Review by ChipWolf — CHANGES_REQUESTED** [review]
The new secret needs to be added at the bottom in valuesFrom and added to the bootstrap-vault script

File: minecraft-dev/minecraft/minecraft-helm-values.txt [hunk 2]
```diff
@@ -3,5 +3,5 @@ extraEnv:
   CFG_MASK_APIKEY: "$MC_MASK_APIKEY"
   CFG_NUVOTE_TOKEN: "$MC_NUVOTE_TOKEN"
   CFG_PIXELMONOPTIMIZER_PASSWORD: "$MC_PIXELMONOPTIMIZER_PASSWORD"
-  CFG_TEBEX_KEY: "$MC_TEBEX_KEY"
+  CFG_TEBEX_KEY: "$MC_TEBEX_CELEBI_KEY"
```
Comment (ChipWolf) [inline minecraft-dev/minecraft/minecraft-helm-values.txt:6]: This change is probably unintentional

**Review by ChipWolf — CHANGES_REQUESTED** [review]

File: minecraft-dev/minecraft/minecraft-helm-values.txt (hunk 2 above)
Comment (LooterToGo) [reply minecraft-dev/minecraft/minecraft-helm-values.txt]: Ya it's unintentional. Blind eyes yesterday.

File: minecraft-dev/minecraft/minecraft-helm-values-celebi.txt (hunk 1 above)
Comment (LooterToGo) [reply minecraft-dev/minecraft/minecraft-helm-values-celebi.txt]: Should be sorted out now. @ChipWolf

File: minecraft-dev/minecraft/minecraft-helm-values.txt (hunk 2 above)
Comment (LooterToGo) [reply minecraft-dev/minecraft/minecraft-helm-values.txt]: Issue resolved.

**Pull request merged**